WootricSDK/WootricSDK/WTRApiClient.m

```diff
@@ -275,6 +296,11 @@ - (void)checkEligibility:(void (^)())eligible {
           if (_settings.forceSurvey) NSLog(@"WootricSDK: forced survey (remove for production!)");
           NSLog(@"WootricSDK: User eligible");
           [_settings parseDataFromSurveyServer:responseJSON];
+          _userID = responseJSON[@"settings"][@"user_id"];
```

Comment:
What will happen here is that `_userID` will be `nil` and it will send `user_id=0` on requests. I will do the same thing I'm doing right now with `account_id` which is that if the value is different than `nil` I add it to the request, if not, I don't. Is that good?